CommunityCall/README.md

```diff
@@ -12,6 +12,7 @@ recurring event on your calendar.
 
 ## Notes
 
+* [November 19, 2020](./notes/20201119_Notes.md) ([YouTube recording](https://youtu.be/IHMgWobHzrI) and [transcript](./notes/20201119_ChatTranscript.txt))
 * [August 20, 2020](./notes/20200820_Notes.md) ([YouTube recording](https://youtu.be/emNWmUcGVq8) and [transcript](./notes/20200820_ChatTranscript.txt))
 * [April 16, 2020](./notes/20200416_Notes.md) ([YouTube recording](https://youtu.be/q3Wx7MCpXnk) and [transcript](./notes/20200416_ChatTranscript.txt))
```

Comment:
Have we a chance to see skipped Calls?

Reply:
Yeah, we're working on getting them all posted still.